### ISSUE TITLE
Detect project for `edgedb instance` commands

### ISSUE THREAD

**Issue by colinhacks** [edgedb/edgedb-cli#750]
Latest nightly seems not to detect projects for `edgedb instance` commands.

Running inside an initialized project:

```
$ edgedb
EdgeDB 2.0-dev.6625+f62dc75 (repl 1.2.0-dev.783+6690d4d)
Type \help for help, \quit to quit.
edgedb> \q
$ edgedb instance stop
edgedb error: Instance name argument is required, use '-I name'
```

**Comment by tailhook** [comment]
Yes. This was the plan: release a version that doesn't support that. So that `edgedb instance stop $VARIABLE` issues a warning (and errors out when `$VARIABLE` is empty). Wait for people to adjust scripts to use `edgedb instance stop -I$VARIABLE` and then detect project.

This is the second part of the #739

**Comment by colinhacks** [comment]
I think it's time to pull the trigger on this @tailhook
